ServerRoomMonitoring.Api/Repository/ISensorRepository.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using ServerRoomMonitoring.Api.Models;
+
+namespace ServerRoomMonitoring.Api.Repository
+{
+    public interface ISensorRepository
+    {
+
+        public List<SensorMessage> GetAllSensors();
```

Comment:
Słowo public przy metodach zbędne. Jak interfejs jest public to wszystkie metody też będą